src/components/Match/renderMatch.js

```diff
@@ -56,44 +56,80 @@ function generateGraphData(match) {
   return {};
 }
 
-function generateTeamfights(match) {
+function generateTeamfights({ players, teamfights = [] }) {
   const computeTfData = (tf) => {
     const newtf = {
       ...tf,
       deaths_pos: [],
+      radiant_networth_advantage_delta: 0,
       radiant_gold_delta: 0,
+      dire_gold_delta: 0,
```

Comment:
Can't we just use radiant_gold_delta?  It's negative if the teamfight was a win for dire.